ocs_ci/ocs/machine.py

```diff
@@ -1117,3 +1118,105 @@ def wait_for_current_replica_count_to_reach_expected_value(
         res = False
 
     return res
+
+
+def delete_machines(machine_names):
+    """
+    Delete the machines
+
+    Args:
+        machine_names (list): List of the machine names you want to delete
+
+    Raises:
+        CommandFailed: In case yaml_file and resource_name wasn't provided
+
+    """
+    machine_obj = OCP(
+        kind="machine", namespace=constants.OPENSHIFT_MACHINE_API_NAMESPACE
+    )
+    for machine_name in machine_names:
+        log.info(f"Deleting machine {machine_name}")
+        machine_obj.delete(resource_name=machine_name)
```

Comment:
why not using https://github.com/red-hat-storage/ocs-ci/blob/9d4c46131f76f58b0cbd830ffab87124e43d9007/ocs_ci/ocs/machine.py#L95 with a for loop from the test itself?

Reply:
Even though it's a small function, I think it's more generic to use the function `delete_machines` and save the extra code here and in the future. Maybe like the function `delete_pvcs`.